post-scan-actions/aws-python-promote-or-quarantine/README.md

```diff
@@ -244,3 +244,32 @@ After a scan occurs, this example Lambda function places clean files in one buck
         - `<YOUR_LAMBDA_FUNCTION_ARN>` is replaced with the Lambda function ARN you found earlier.
     </details>
 
+## Test the Quarantine Function
+
+1. **Download the Eicar test file**
+   - The Eicar test virus can be downloaded here: `https://secure.eicar.org/eicar.com`
+2. **Upload the Eicar file to the scanning bucket**
+
+    <details>
+    <summary>Using AWS Console</summary>
+
+    1. Go to **Services > S3**.
+    2. Search for your scanning bucket.
+    3. Click on the **Upload** button. 
+    4. Find the Eicar test file you downloaded and click **Upload**.
+    5. Check your quarantine bucket to see if the Eicar file has moved.
+    **NOTE:** It can take about 30-60 seconds or more for the file to move.
+
+    </details>
+
+    <details>
+    <summary>Using the AWS CLI</summary>
+
+    - Enter the folowing AWS CLI command to upload the Eicar test file to the scanning bucket:
+
+        `aws s3 cp eicar.com s3://<YOUR_SCANNING_BUCKET> `
+    - where:
+        - `<YOUR_SCANNING_BUCKET>` is replaced with the S3 scanning bucket name.
+
+    **NOTE:** It can take about 30-60 seconds or more for the file to move.
```

Comment:
```suggestion
    📌 It can take 30-60 seconds or more for the 'move' operation to complete, and during this time, you may see the file in both buckets.
```

Reply:
Suggestions applied LGTM.